src/package-request.js

```diff
@@ -5,6 +5,12 @@ import type PackageResolver from './package-resolver.js';
 import type {Reporter} from './reporters/index.js';
 import type Config from './config.js';
 import type {Install} from './cli/commands/install';
+
+import path from 'path';
+
+import invariant from 'invariant';
+import semver from 'semver';
```

Comment:
Be careful with this, I think some of our third-party modules (or their dependencies) are sometimes mocked in our tests, so using import will break them. Case in mind: `request`.